package.json

```diff
@@ -112,7 +110,23 @@
     "eslint-plugin-react": "^7.32.2",
     "msw": "^1.0.0",
     "prettier": "^2.7.1",
+    "snyk": "^1.1293.1",
```

Comment:
Snyk needs to be part of packages included in the MC app?

Reply:
if If we want to run it locally, then yes. I can remove it, but with the next operations, Snyk will probably install it again.

Reply:
We need to better identify how we want to make use of Snyk. I think we should leave it out of the submitted changes for now, until we get a better grasp on it.